rpc/src/rpc_service.rs

```diff
@@ -97,6 +101,7 @@ impl RpcRequestMiddleware {
             )
             .unwrap(),
             snapshot_config,
+            identity_info_map: Arc::new(map_vote_identity_to_info(&bank_forks, &vote_accounts_to_monitor)),
```

Comment:
I wonder, is `bank_forks` already populated with banks this early in the startup sequence? I suspect so, loading the bank is probably why validator startup takes so long in the first place ... but it would be good to confirm, have you tested this?

I think we could test with a `solana-test-validator`, create a new vote account there and set the info, then restart the test validator and monitor that vote account.

Reply:
I have tested with local cluster using `./multinode-demo/bootstrap-validator.sh` and `./multinode-demo/validator-x.sh` from Solana repo. Did multiple restarts in the process.

Reply:
:+1: LGTM then!